site/src/components/image.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Image
+description: Image component in the Atlas Design System
+template: standard
+---
+
+# Image
+
+The image component parents an image tag or svg and provides width based classes to contrain its children.
+
+## Usage
+
+Here is an example of a standard usage of label paired with an input. The default image class takes up the full width of its container.
```

Comment:
Remove the first sentence :)

Reply:
Guess which file I copied this from ... :D

Reply:
updated